woocommerce/woocommerce-gutenberg-products-block
# Rename Checkbox List story to work with storybook 7

## What

Renames Checkbox List story to work with storybook 7

## Why

Previous PR #11469 was merged but does not work with storybook 7.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Run `npm run storybook`
2. Go to `External Components -> Checkbox List` and ensure it works.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
